ouroboros-consensus/src/Ouroboros/Consensus/Node/ErrorPolicy.hs

```diff
@@ -1,13 +1,19 @@
-{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE PartialTypeSignatures #-}
```

Comment:
As much as I appreciate this extension (and its author :wink:), it's not needed here.